### PR TITLE
Added: JCG latest tourney retriever, returns tourney code

### PR DESCRIPTION
**Pull request by YahikoSV**

### CODE REVIEW

File: website_scraper.py [hunk 1]
```diff
@@ -27,6 +27,55 @@ def SVO_initial_scraper(svoexcel):
...
+        #Does it meet our conditions? (Note: First one is always the latest so no dates were used for now)
+        if potential_format == formats[sv_format] and potential_stage == stage[tourney_stage] and potential_status == '終了':
+            latest_tourney = True
+            potential_id = tourney_code.get("competition_id")
+            tourney_date = tourney_code.find('td', class_="date").text
+            latest_tourney_code = str(potential_id)
+            #latest_jsonlink = f'https://sv.j-cg.com/compe/view/entrylist/{latest_tourney_code}/json'
+            break
+       
+    if latest_tourney == True:
```
Comment (Zhiff) [inline website_scraper.py]: Print function is not required, this can be removed

Comment (Zhiff) [reply website_scraper.py]: but i guess its fine

File: website_scraper.py [hunk 2]
```diff
@@ -27,6 +27,55 @@ def SVO_initial_scraper(svoexcel):
...
+        print(f'Tourney Code: {latest_tourney_code} Date: {tourney_date}') #verification
+    else:
+        print('No tourney found')
+    
+    #return latest_tourney_code
```
Comment (Zhiff) [inline website_scraper.py]: set latest_tourney value to None if the value is not found at all

Comment (Zhiff) [reply website_scraper.py]: uncomment this line after  latest_tourney value setting to None is added, since other than that everything seems fine

**Review by Zhiff — CHANGES_REQUESTED** [review]
Remove the cache from this commit, they are not needed in repository.

**Comment by YahikoSV** [comment]
So I close this pull request and open another one.

**Pull request merged**